test/mailers/mentor_notifications_mailer_test.rb

```diff
@@ -63,6 +63,9 @@ class MentorNotificationsMailerTest < ActionMailer::TestCase
     str = "A student you are mentoring (#{user.handle}) has posted"
     assert_body_includes email, str
     assert_text_includes email, str
+
+    link_to_iteration = Rails.application.routes.url_helpers.my_solution_iteration_url(iteration.solution, iteration)
```

Comment:
I've not validated this but I don't believe that you need the `Rails.application.routes.url_helpers.` part of this.